assemblies/assembly-planner-configuration.adoc

```diff
@@ -8,9 +8,9 @@
 
 Solving a planning problem with OptaPlanner consists of the following steps:
 
-. *Model your planning problem* as a class annotated with the `@PlanningSolution` annotation (for example, the `NQueens` class).
-. *Configure a Solver* (for example a First Fit and Tabu Search solver for any `NQueens` instance).
-. *Load a problem data set* from your data layer (for example a Four Queens instance). That is the planning problem.
+. *Model your planning problem* as a class annotated with the `@PlanningSolution` annotation (for example, the `TimeTable` class).
+. *Configure a Solver* (for example a First Fit and Tabu Search solver for any `TimeTable` instance).
+. *Load a problem data set* from your data layer (for example a timetable instance). That is the planning problem.
```

Comment:
(for example a `Timetable` instance with populated timeslotList, roomList, lessonList data)